airflow/www_rbac/static/bootstrap-toggle.min.css

```diff
@@ -0,0 +1,28 @@
+/*! ========================================================================
```

Comment:
Would it be possible to point to the static assets from www/ instead of having to duplicate them in this tree?

Reply:
At the start I wasn't familiar with the amount of modification required, so it was easier to copy/paste the entire folder and not worry about breaking old UI (this will also make deprecating the older webserver easier).  In the end, maybe 1-2 static files were slightly modified, most untouched. so I don't feel too strong about going either way.